src/org/labkey/test/components/experiment/UserActivateDeactivateConfirmationDialog.java

```diff
@@ -0,0 +1,54 @@
+package org.labkey.test.components.experiment;
+
+import org.labkey.test.WebDriverWrapper;
+import org.labkey.test.components.bootstrap.ModalDialog;
+import org.labkey.test.pages.LabKeyPage;
+
+import java.util.function.Supplier;
+
+public class UserActivateDeactivateConfirmationDialog<SourcePage extends WebDriverWrapper, ConfirmPage extends LabKeyPage> extends ModalDialog
```

Comment:
I'm not really sure I understand what is added by creating this class, couldn't you use the ModalDialog as is?  For example in my tasks test I have this code:
        ModalDialog dialog = tasksPage.getJobProgressPanel().deleteTask();
	....
        dialog.dismiss("Yes, Delete");

Where getJobProgressPanel().deleteTask() is:
    public ModalDialog deleteTask()
    {
	...
        return new ModalDialog.ModalDialogFinder(getDriver())
                .withTitle("Permanently delete task?").timeout(500).waitFor();
    }

Couldn't the userAdminPage do the something similar?